RELEASING.txt

```diff
@@ -0,0 +1,24 @@
+We use zest.releaser for release automation.
+
+https://pypi.org/project/zest.releaser/
+
+To make a new release, first prepare by adding any omitted entries to
+CHANGES.rst. Then use the following command, accepting all default options:
```

Comment:
I actually don't know if this is true. :wink:

I assume that when you actually run through it, you will make notes for any exceptions to the defaults.